assets/cubyz/recipes/brick_recipes.zig.zon

```diff
@@ -23,6 +23,10 @@
 		.inputs = .{"cubyz:glacite"},
 		.output = "2 cubyz:glacite_wall",
 	},
+	.{
+		.inputs = .{"cubyz:stone"},
+		.output = "cubyz:cobblestone,
```

Comment:
Please test changes like this next time. You are missing the closing parenthesis `"` here.